Source/DafnyDriver/Commands/VerifyCommand.cs

```diff
@@ -28,6 +36,13 @@ public static class VerifyCommand {
   public static readonly Option<string> FilterPosition = new("--filter-position",
     @"Filter what gets verified based on a source location. The location is specified as a file path suffix, optionally followed by a colon and a line number. For example, `dafny verify dfyconfig.toml --filter-position=source1.dfy:5` will only verify things that range over line 5 in the file `source1.dfy`. In combination with `--isolate-assertions`, individual assertions can be verified by filtering on the line that contains them. When processing a single file, the filename can be skipped, for example: `dafny verify MyFile.dfy --filter-position=:23`");
 
+  public static readonly Option<string> ExtractTarget = new("--extract", @"
```

Comment:
I think I'd prefer for this to be a top-level command, and perhaps a little more explanatory. One possibility would be `dafny extract-boogie-axioms`. (The types and functions are missing definitions, and are essentially axioms, as well.)

Reply:
One possible downside is that I'm not sure whether commands can be hidden...

Reply:
I, too, would like to see this as a top-level command. However, since I don't understand how commands are added to the source or how we make a command do certain things, I'm leaving this as is for now. I will ask @keyboardDrummer for help to make such a change in a separate PR.